app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+class CommandError(Exception):
+    pass
+
+
+def copy_file(command: str) -> None:
+    try:
+        command = command.split()
+        if len(command) != 3:
+            raise CommandError("Unsupported format of command")
+
+        cmd, file_name, new_file_name = command
+        if cmd == "cp" and file_name != new_file_name:
+            with (open(file_name, "r") as source,
+                    open(new_file_name, "w") as new_file):
+                new_file.write(source.read())
+            print("Copying done!")
+
+        raise CommandError(f"Unknown command: {command[0]}\n"
```

Comment:
You need to change this, error is raising both on `cmd == "cp"` this check and on `file_name != new_file_name` this check and even if code finish correctly